pkg/apis/tensorflow/v1alpha1/register.go

```diff
@@ -11,8 +11,14 @@ var (
 	AddToScheme   = SchemeBuilder.AddToScheme
 )
 
-// GroupName is the group name use in this package
-const GroupName = "tensorflow.org"
+const (
+	// GroupName is the group name use in this package.
+	GroupName = "tensorflow.org"
```

Comment:
nit: s/tensorflow.org/kubeflow.org
@jlewi WDYT?

Reply:
I don't have a strong opinion. Feel free to open an issue.